### PR TITLE
Remove unecessary bindActionCreator calls

### PR DESCRIPTION
**Pull request by jasonLaster**
### Summary of Changes

Quick refactor inspired by @Fischer-L's work cleaning up our connect calls.

### CODE REVIEW

**Comment by jasonLaster** [comment]
Hmm, looks like this approach doesn't work without bindActionCreators... Maybe i'm missing something.

CC @wldcordeiro @Fischer-L

**Comment by wldcordeiro** [comment]
@jasonLaster I think the object that `actions` is, isn't a normal JS object so Redux doesn't bind properly. 🤔

**Comment by jasonLaster** [comment]
ooh interesting... it should just come the actions/index module. 

Worth investigating :8ball:

File: src/components/Editor/GutterMenu.js [hunk 1]
```diff
@@ -154,19 +153,16 @@ class GutterContextMenuComponent extends PureComponent {
...
-      pauseData: getPause(state),
-      contextMenu: getContextMenu(state),
-      emptyLines: selectedSource
-        ? getEmptyLines(state, selectedSource.toJS())
-        : []
-    };
-  },
-  dispatch => bindActionCreators(actions, dispatch)
-)(GutterContextMenuComponent);
+export default connect(state => {
```
Comment (lukaszsobek) [inline src/components/Editor/GutterMenu.js]: This seems very hard to read, maybe we just make a mapStateToProps and mapDispatchToProps function to handle the logic. 1.) It is the default way of handling these kinds of things and 2.) will make it easier to see what is going on and also 3.) eliminate mistakes due to nested bracketing.

Comment (jasonLaster) [reply src/components/Editor/GutterMenu.js]: I'd like to figure out how to have `connect(mapStateToProps, actions)(GutterContextMenuComponent);` everywhere

`

**Comment by jasonLaster** [comment]
closing for now

**Pull request closed without merging**